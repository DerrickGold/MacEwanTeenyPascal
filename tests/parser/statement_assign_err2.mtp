begin
	(*Bad Assignment*)

	(*assignment statement: with factor (constant)*)
	something : 10;
end
.