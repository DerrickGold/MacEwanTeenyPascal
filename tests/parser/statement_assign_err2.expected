Syntax Error: line 5:
Unexpected token (TOK_COLON, ':'), expected a token type of:  
  'TOK_ASSIGN' or
  'TOK_LPAREN'
